### ISSUE TITLE
IMPORTANT - Using alt

### ISSUE THREAD

**Issue by Emik234** [eja0623/prj-rev-bwfs-dasmoto#3]
You should consider adding an alt attribute to the images, that way if any of the images don't load for some reason, you don't get the broken link icon, rather the text you specify in alt will be displayed, more info here: https://www.w3schools.com/tags/att_img_alt.asp

**Comment by eja0623** [comment]
Good point.  It ties in with your other point about downloading the images and using a relative path to link them to. I actually thought about resubmitting the project with the images in their own folder, in case the link from the absolute path gets disrupted somehow. I'll probably go back and do that just for practice. 
Thanks for the feedback, and resources.  I'll check out the links you provided. All good info.
